app/controllers/admin/resources_controller.rb

```diff
@@ -48,5 +56,24 @@ def destroy
     def resource_params
       params.require(:resource).permit(:name, :description, :quantity, :used, :conference_id)
     end
+
+    def check_successful_update(increment_used_resource_flag)
```

Comment:
This is not needed if you just have params[:used_change] which has values of -1 or +1